apis/elbv2/v1beta1/ingressclassparams_types.go

```diff
@@ -41,6 +41,15 @@ const (
 	LoadBalancerSchemeInternetFacing LoadBalancerScheme = "internet-facing"
 )
 
+// SubnetSelector selects one or more existing subnets.
+type SubnetSelector struct {
+	// ID specifies the resource ID of a subnet. Exactly one of this or `nameContains` must be specified.
+	ID string `json:"id,omitempty"`
+	// NameContains specifies all subnets containing a string in their Name tag. Exactly one of this or `id` must be specified.
+	NameContains string `json:"nameContains,omitempty"`
```

Comment:
There is a possibility of multiple matches here and we need a strategy to break ties in case of multiple subnets per AZ. In case of subnet name/id in the annotation, controller currently expects all of the entries to be resolved to valid subnets, and no multiples in the same AZ.

Reply:
I was thinking we'd have the same expectation; that the union of all `SubnetSelector`s contains at most one subnet per AZ. It is true that the situation could change between the time the `IngressClassParams` is last modified and the LB is created or modified.

Reply:
Apparently the annotation expects exactly one match. I was expecting this to handle multiple matches, but still at most one subnet per AZ.

Reply:
I think devising rules for multiple subnets in an AZ would be for a separate PR.